### PR TITLE
Make modifications across codebase to conform with flake8

### PR DESCRIPTION
**Pull request by exhaustin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: src/home_robot/home_robot/utils/data_tools/robohive.py [hunk 1]
```diff
@@ -187,7 +188,7 @@ def main(
     # plot paths
     if plot_paths:
         file_name = output_dir + "/" + output_name + "{}".format(time_stamp)
-        plotnsave_paths(paths, env=env, fileName_prefix=file_name)
+        plotnsave_paths(plot_paths, env=env, fileName_prefix=file_name)
```
Comment (cpaxton) [inline src/home_robot/home_robot/utils/data_tools/robohive.py:191]: we may want to delete this robohive file  - @zephirefaith  any thoughts

File: src/home_robot/home_robot/utils/data_tools/writer.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def __init__(self, filename="data.h5", dirname=None):
         if dirname is not None:
             try:
                 os.mkdir(dirname)
-            except OSError as e:
+            except OSError:
```
Comment (cpaxton) [inline src/home_robot/home_robot/utils/data_tools/writer.py:33]: should we print this error instead of deleting it?

File: src/home_robot_hw/home_robot_hw/ros/camera.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def __init__(
         self.buffer_size = buffer_size
         if self.buffer_size is not None:
             # create buffer
-            self._buffer = deque()
+            self._buffer: deque = deque()
```
Comment (cpaxton) [inline src/home_robot_hw/home_robot_hw/ros/camera.py:50]: I dont understand this change

File: src/home_robot_hw/home_robot_hw/ros/grasp_helper.py [hunk 4]
```diff
@@ -80,7 +80,7 @@ def _cb(self, msg):
         # Convert rospy point cloud into a message that we care about
         with self.grasp_lock:
             self.grasp_id = msg.seq
-            self.poses = msg_to_poses(msg)
+            self.poses = self.msg_to_poses(msg)
```
Comment (cpaxton) [inline src/home_robot_hw/home_robot_hw/ros/grasp_helper.py:83]: what was happening before? how was this working?

File: src/home_robot_hw/home_robot_hw/ros/grasp_helper.py [hunk 5]
```diff
@@ -104,19 +104,7 @@ def msg_to_poses(self, msg, frame=None):
     def _score_cb(self, msg):
         with self.score_lock:
             self.score_id = msg.header.seq
-            self.scores = np.array([pt.x for x in msg.polygon.points])
-
-    def segmented_point_cloud_to_msg(self, xyz, labels):
```
Comment (cpaxton) [inline src/home_robot_hw/home_robot_hw/ros/grasp_helper.py:109]: why is this deleted?

File: src/home_robot_hw/home_robot_hw/ros/msg_numpy.py [hunk 6]
```diff
@@ -101,55 +102,13 @@ def image_to_numpy(msg):
 
 # @converts_from_numpy(Image)
 def numpy_to_image(arr, encoding):
-    if not encoding in name_to_dtypes:
```
Comment (cpaxton) [inline src/home_robot_hw/home_robot_hw/ros/msg_numpy.py:104]: again this is a weird amount of code getting deleted in a refactor - what's happening here?

**Review by cpaxton — COMMENTED** [review]
Little concerned about merging this many ROS changes without real robot tests

File: src/home_robot_hw/home_robot_hw/ros/grasp_helper.py (hunk 5 above)
Comment (exhaustin) [reply src/home_robot_hw/home_robot_hw/ros/grasp_helper.py:109]: Two `segmented_point_cloud_to_msg` were defined in the same class. Only one seems to be used (the one with both rgb and xyz as inputs), so I removed the other one.

File: src/home_robot_hw/home_robot_hw/ros/grasp_helper.py (hunk 4 above)
Comment (exhaustin) [reply src/home_robot_hw/home_robot_hw/ros/grasp_helper.py:83]: There's no way this was working before. We probably just never used this function.

File: src/home_robot_hw/home_robot_hw/ros/camera.py (hunk 3 above)
Comment (exhaustin) [reply src/home_robot_hw/home_robot_hw/ros/camera.py:50]: This is just typing. Mypy was complaining about it, but that was before I gave up on enabling it for now so we can remove this change.

File: src/home_robot_hw/home_robot_hw/ros/msg_numpy.py (hunk 6 above)
Comment (exhaustin) [reply src/home_robot_hw/home_robot_hw/ros/msg_numpy.py:104]: The exact same `numpy_to_image` function was declared two times -- probably a mistake during copy-paste.

**Comment by cpaxton** [comment]
Real robots tests passed
